starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,78 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+}
 
-// }
+Chronometer.prototype.startClick = function () {
+  // because of scoping, make a copy of this (otherwise we can't access this.currentTime++;)
+  var self = this;
+  self.intervalId = setInterval(function () {
+    self.currentTime++;
+    self.setTime();
+  }, 10);
+};
 
-// Chronometer.prototype.startClick = function () {
+Chronometer.prototype.setMinutes = function () {
+  // return how many minutes (from seconds total). (10 * 6000 = 60.000 miliseconds which is 1 minute)
+  return Math.floor(this.currentTime / 6000);
+};
 
-// };
+Chronometer.prototype.setSeconds = function () {
+  // return seconds left after calculating minutes. 
+  return Math.floor((this.currentTime % 6000) / 100);
+};
 
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
+Chronometer.prototype.setMilliseconds = function () {
+  // current time 100 (10*100) = 1 second. return what's left.
+  return this.currentTime % 100;
+};
 
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
+Chronometer.prototype.twoDigitsNumber = function (number) {
+  // if one digit, add a leading 0
+  if (number < 10) {
+    return "0" + number.toString();
+  } else {
+    return number.toString();
+  }
+};
```

Comment:
Since you seem to have an appreciation for concise code you might be interested in using [ternary operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) for simple conditional statements: 
```
return (number < 10) ? "0" + number.toString() :  number.toString()
```